Makefile

```diff
@@ -123,22 +130,20 @@ endif
 		.
 
 .PHONY: release-tester
-release-tester: dist/$(DOCKERFILE) ## Build image used for testing a specific release package
+release-tester: dist/$(TESTDOCKERFILE) ## Build image used for testing a specific release package
```

Comment:
why the dependency on `dist/$(TESTDOCKERFILE)` those are committed, don't need to be built?

Reply:
The docker command which is invoked by this stage does depend on the contents of the docker file. When testing this locally (and making changes to the dockerfile), one would want it to rebuild.